scripts/scriptutils/conversions.py

```diff
@@ -38,12 +43,14 @@ def convert_package_sbol2_files(package: str) -> dict[str, str]:
             merge_doc = sbol3.Document()
             merge_doc.read(file3)
             # figure out which ones to copy
-            non_replaced = set(o.identity for o in merge_doc.objects) - set(o.identity for o in doc3.objects)
+            non_replaced = set(o.identity for o in merge_doc.objects) - set(
+                o.identity for o in doc3.objects
+            )
```

Comment:
What is the rule that's doing this refactor? If feels strange to have the change applied to the second set expression but not the first.

Reply:
That's a good question, this has been done by black. Line-length > 99 when taking into account the indent ? But I don't know why it did not get it consistent across the full expression.